tests/test_game.py

```diff
@@ -284,3 +284,8 @@ def test_json_accepts_any_order_of_params(self):
         ttt_json = TicTacToe.from_json(**json_dict)
 
         assert torch.all(ttt.board_state.eq(ttt_json.board_state))
+
+    def test_tie_board_is_game_over(self):
+        board = torch.Tensor([[1, 0, 1], [1, 0, 1], [0, 1, 0]])
+
+        assert TicTacToe.is_game_over(board)
```

Comment:
can we clarify that this assert should be true? I find it more readable (despite being crappy efficiency practice)